tests/test_crawl.py

```diff
@@ -23,6 +23,12 @@ def setUp(self):
     def tearDown(self):
         self.mockserver.__exit__(None, None, None)
 
+    @defer.inlineCallbacks
+    def test_callback_kwargs(self):
+        crawler = self.runner.create_crawler(KeywordArgumentsSpider)
+        yield crawler.crawl(mockserver=self.mockserver)
+        self.assertEqual(crawler.spider.checks, set([True]))
```

Comment:
Not sure if this is the best way to test this, please advice if it's not.

Reply:
They look fine for me, just not sure if testing a set with only one true value.
What do you think of a list? That way we can ensure that there `n` true elements to make sure all assertion were checked.

Reply:
That makes sense, it would be useful to verify the amount of checks. Thanks for the suggestion :-)